src/react-components/code-sample.tsx

```diff
@@ -33,59 +41,79 @@ export type CodeSamplesViewerProps = {
    * the component. Receives data about the selected code sample. The library
    * comes pre-packaged with some sensible options.
    *
-   * @see CodeSampleMethodTitle
+   * @see CodeSampleTitle
    * @see CodeSampleFilenameTitle
    * @default CodeSampleMethodTitle
    */
-  title?: CodeSampleTitleComponent | React.ReactNode | string;
-  /** The operation to get a code sample for. Can be queried by either
-   * operationId or method+path.
+  title?: CodeSampleTitleComponent | React.ReactNode | string | "none";
```

Comment:
more of a question - would it be better to use `false` here in lieu of `"none"`, since `"none"` is also a string, which we attempt to render below?

 I can't imagine a user would ever intentionally pass `"none"` as a title though. Feel free to ignore this comment.

Reply:
Oh ya, i like that